packages/gestalt/src/Link/InternalLink.tsx

```diff
@@ -141,28 +141,30 @@ const InternalLinkWithForwardRef = forwardRef<HTMLAnchorElement, Props>(function
       ? {
           [layoutStyles.inlineFlex]: !fullWidth,
           [layoutStyles.flex]: fullWidth,
-          [layoutStyles.justifyCenter]: true,
```

Comment:
can we do here

    isButtonSocial && !isInVRExperiment



    isButtonSocial && isInVRExperiment
    
    
    They are significantly different, I think we can benefit from separating concerns. Ok to duplicate classes.